src/sagemaker/model_card/model_card.py

```diff
@@ -208,6 +212,458 @@ def search_model_associated_model_cards(model_id: str):
+            approval_description (str, optional): A description provided for the model approval (default: None).
+            model_approval_status (ModelApprovalStatusEnum or str, optional): Current approval status of model package (default: None).
+            model_package_group_name (str, optional): If the model is a versioned model, the name of the model group that the versioned model belongs to (default: None).
+            model_package_name (str, optional): Name of the model package (default: None).
+            model_package_version (int, optional): Version of the model package (default: None).
+            domain (str, optional): The machine learning domain of the model package you specified. Common machine learning domains include computer vision and natural language processing (default: None).
+            task (str, optional): The machine learning task you specified that your model package accomplishes. Common machine learning tasks include object detection and image classification (default: None).
+            created_by (ModelPackageCreator, optional): Information about the user who created model package (default: None).
+            source_algorithms (SourceAlgorithms, optional):A list of algorithms that were used to create a model package (default: None).
+            inference_specification (InferenceSpecification, optional): Details about inference jobs that can be run with models based on this model package (default: None).
+            model_metrics (dict, optional): Temporary field to store model metrics that will be usd to auto discover evaluation details (default: None).
+        """  # noqa E501 # pylint: disable=line-too-long
+        self.model_package_arn = model_package_arn
+        self.model_package_description = model_package_description
+        self.model_package_status = model_package_status
+        self.model_approval_status = model_approval_status
+        self.approval_description = approval_description
+        self.model_package_group_name = model_package_group_name
+        self.model_package_name = model_package_name
+        self.model_package_version = model_package_version
+        self.domain = domain
+        self.task = task
+        self.created_by = created_by
+        self.source_algorithms = source_algorithms
+        self.inference_specification = inference_specification
+        self.model_metrics = model_metrics
+
+    @staticmethod
+    def call_describe_model_package(model_package_name: str, sagemaker_session: Session = None):
+        """Load existing model package.
+
+        Args:
+            model_package_name (str): Model package ARN.
+            sagemaker_session (Session, optional): A SageMaker Session
+                object, used for SageMaker interactions (default: None). If not
+                specified, a SageMaker Session is created using the default AWS configuration
+                chain.
```

Comment:
nit: add raises

Reply:
What do you mean by `raises` here? I am unable to understand. Are you suggesting to add `raise Warning(....)` type of message if session is not specified?